docs/release-notes/8.19.asciidoc

```diff
@@ -1,6 +1,29 @@
 [[release-notes-header-8.19.0]]
 == 8.19
 
+[discrete]
+[[release-notes-8.19.9]]
+=== 8.19.9
+
+[discrete]
+[[features-8.19.9]]
+==== New features
+* Hardens {elastic-defend} against cloud filter rebinding.
```

Comment:
```suggestion
* Hardens {elastic-defend} against cloud filter rebinding.
```
This attribute isn't rendering properly in the preview, which is weird because the same attribute is rendering elsewhere 😕

Reply:
I _think_ it's fine: https://security-docs_bk_7120.docs-preview.app.elstc.co/guide/en/security/current/release-notes-header-8.19.0.html#features-8.19.9